src/modules/75-cf/pages/feature-flags/components/FeatureFlagsListing.tsx

```diff
@@ -102,6 +117,33 @@ const FeatureFlagsListing: FC<FeatureFlagsListingProps> = ({
 
   const columns: Column<Feature>[] = useMemo(
     () => [
+      {
+        id: 'staleCheckbox',
+        Header: (
+          <Checkbox
+            aria-label="selectAllFlags"
+            onClick={() => {
+              //TOD: Handle Select all
+            }}
+          />
```

Comment:
You could do with a wrapping tag here. As-is the table will stretch this component to fill the space
```suggestion
          <div>
            <Checkbox
              aria-label="selectAllFlags"
              onClick={() => {
                //TOD: Handle Select all
              }}
            />
          </div>
```

Reply:
noted, thank you